stratum/hal/lib/tdi/es2k/es2k.bzl

```diff
@@ -0,0 +1,35 @@
+# Copyright 2018-present Open Networking Foundation
+# Copyright 2022 Intel Corporation
+# SPDX-License-Identifier: Apache-2.0
+
+# This Starlark rule imports the ES2K SDE shared libraries and headers.
+# The SDE_INSTALL or SDE_INSTALL_TAR environment variable needs to be set;
+# otherwise, the Stratum rules for ES2K platforms cannot be built.
+
+def _impl(repository_ctx):
```

Comment:
Change environment variable names to `ES2K_INSTALL` / `ES2K_INSTALL_TAR`. The Stratum SDK detection mechanism depends on each package having a unique environment variable name.